__init__.py

```diff
@@ -133,8 +133,21 @@ def currency_filter(num, grouping=True):
 def date_filter_sm(date_str):
     if not date_str:
         return ''
-    return parse_date(date_str).strftime('%m%y')
+    return parse_date(date_str).strftime('%m/%y')
 
+@app.template_filter('date_md')
+def date_filter_md(date_str):
+    if not date_str:
+        return ''
+    return parse_date(date_str).strftime('%b %Y')
+
+@app.template_filter('last_3_characters')
+def last_3_characters(int):
```

Comment:
How about a more concise implementation, e.g.
```python
def last_n_characters(value, nchar=3):
    if type(value) == int:
        return int % (10 ** nchar)
    return ''
```
If `nchar` is parameterized (I defaulted it to `3` above), then we can ask for arbitrary numbers of digits if we ever want to.